deploy/017-deploy-vai-controller.ts

```diff
@@ -0,0 +1,53 @@
+import { ethers } from "hardhat";
+import { DeployResult } from "hardhat-deploy/dist/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { getConfig } from "../helpers/deploymentConfig";
+import { toAddress } from "../helpers/deploymentUtils";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const deploymentConfig = await getConfig(hre.network.name);
+  const { preconfiguredAddresses } = deploymentConfig;
+
+  const accessControlManagerAddress = await toAddress(
+    preconfiguredAddresses.AccessControlManager || "AccessControlManager",
+    hre,
+  );
+
+  // VAIController Beacon
+  const vaiControllerImp: DeployResult = await deploy("VAIControllerImp", {
```

Comment:
Nit: I think we use `Impl` for Comptroller and VToken, maybe we could do the same here for consistency

```suggestion
  const vaiControllerImp: DeployResult = await deploy("VAIControllerImpl", {
```